www/components/Hr/Hr.js

```diff
@@ -0,0 +1,7 @@
+import styles from './hr.module.css';
+
+const Hr = () => {
+  return <hr className={styles.hr} />;
+};
+
+export default Hr;
```

Comment:
Can you explain this? Why not throw a border onto existing elements?

Reply:
It's just a dividing line. We could just put a border for sure but sometimes it is better to create a separate element to make it clearer where the border is coming from in the markup.